### ISSUE TITLE
"Sign in to continue" dialog popping up after Dynamics 365 CE Wave 2 2022 update

### ISSUE THREAD

**Issue by ZiaRahman1** [Microsoft/EasyRepro#1320]
## Question

I updated our sandbox to 2022 Wave 2 update. When I tried to run my automation, I get this popup again and again and I can't run the automation. I ran the automation on different sandbox that was not updated and it I didn't see this issue. This happens in chrome incognito mode.
![image](https://user-images.githubusercontent.com/11154264/190496601-6782406f-acda-4f5c-86ad-fc1d9f74d3b5.png)
I search online and found one reddit link, nothing else. here is reddit link https://www.reddit.com/r/Dynamics365/comments/w5jyr9/some_components_of_this_app_require_you_to_sign/

Anyone have seen this issue? Any suggestions?

**Comment by RaviYenumula** [comment]
Yes one of my colleague mentioned he come across the same error after Wave 2 update. I didn't get chance to investigate yet but I will find more details about it. 
Worth trying the following options 
1) check if it is same issue in both incognito mode and normal mode
2) try enable the cookies and see if that helps any thing. 

Sorry not much helpful but I will come back to as soon as I get an access to the instance where I can reproduce this.

**Comment by ZiaRahman1** [comment]
Thanks @RaviYenumula for your suggestions.  Cookie settings did the trick. I set the CookieСontrolsMode to "0" in browserOptions.cs files.

**Comment by jidmadhu** [comment]
Thanks @ZiaRahman & @RaviYenumula . This really helped!

**Comment by arthurgalanti** [comment]
> Thanks @RaviYenumula for your suggestions. Cookie settings did the trick. I set the CookieСontrolsMode to "0" in browserOptions.cs files.

Thanks, this works
